openverse_catalog/dags/common/loader/provider_details.py

```diff
@@ -121,7 +121,6 @@ class ImageCategory(Enum):
 
 # Default image category by source
 DEFAULT_IMAGE_CATEGORY = {
-    "flickr": ImageCategory.PHOTOGRAPH.value,
```

Comment:
Did you mention why this is being removed? As I understand Flickr is exclusively photos (and some videos) and not illustrations or other media types.

Reply:
Flickr returns one of 3 `type` values, and we set the category as `photograph` only to the `photos`, not `screenshots` or `other`:
```python
def _get_category(image_data):
    """
    Flickr has three types:
        0 for photos
        1 for screenshots
        3 for other
    Treating everything different from photos as unknown.
    """
    if "content_type" in image_data and image_data["content_type"] == "0":
        return ImageCategory.PHOTOGRAPH.value
    return None
```
So, even before this PR, Flickr script was using `photograph` as the category for type 0. I don't know if we should investigate what types 1 and 3 are, and whether we can use them for categorizing as well.